source/client/process_impl.cc

```diff
@@ -518,23 +535,36 @@ std::chrono::nanoseconds ProcessImpl::computeInterWorkerDelay(const uint32_t con
   return std::chrono::duration_cast<std::chrono::nanoseconds>(inter_worker_delay_usec * 1us);
 }
 
-void ProcessImpl::createWorkers(const uint32_t concurrency,
-                                const absl::optional<Envoy::SystemTime>& scheduled_start) {
+absl::Status ProcessImpl::createWorkers(const uint32_t concurrency,
+                                        const absl::optional<Envoy::SystemTime>& scheduled_start) {
   ASSERT(workers_.empty());
   const Envoy::MonotonicTime first_worker_start =
       computeFirstWorkerStart(time_system_, scheduled_start, concurrency);
   const std::chrono::nanoseconds inter_worker_delay =
       computeInterWorkerDelay(concurrency, options_.requestsPerSecond());
   int worker_number = 0;
   while (workers_.size() < concurrency) {
+    absl::StatusOr<std::vector<UserDefinedOutputPluginPtr>> plugins =
+        createUserDefinedOutputPlugins(user_defined_output_factories_, worker_number);
+    if (!plugins.ok()) {
+      ENVOY_LOG(error, absl::StrCat("Received error while starting user defined output factories: ",
```

Comment:
Nit: Can we use placeholder syntax instead? i.e. ENVOY_LOG(error, "Received error while starting user defined output factories: {}", plugins.status().message());

Reply:
While working on this, realized that the function it's calling isn't actually using StatusOr anymore, so I'm just removing this conditional.